File: scripts/commands/generic/cmdSetSpeed.py
import sys
from resources.player import AccessLevel

def execute(objManager, player, target, args):
	ghost = player.getCreatureObject()
	
	if player.getAccessLevel() == AccessLevel.PLAYER:
		print("Error: Your Accesslevel is to low to use that command")
		return	
	
	if ghost is None:
		print("Error: No Player or CreatureObject")
		return
		
	ghost.setMovementScale(float(args))
	return

File: scripts/commands/generic/cmdSetSpeed.js
var execute = function(objManager, player, target, args) {
	var creature = player.getCreatureObject();
	var AccessLevel = Java.type("resources.player.AccessLevel");
	
	if(player.getAccessLevel() == AccessLevel.PLAYER) {
		print("Error: Your Accesslevel is to low to use that command");
		return;
	}
	
	if(creature == null) {
		print("Error: No Player or CreatureObject");
		return;
	}
	
	creature.setMovementScale(args);
}
